art/art.py

```diff
@@ -437,7 +437,7 @@ def __word2art(word, font, chr_ignore, letters, next_word):
     splitter = "\n"
     if "win32" != sys.platform:
         splitter = "\r\n"
-    if len(word) == 0:
+    if len(word) == 0 and next_word:
         return splitter
```

Comment:
Wouldn't it better if we return "" instead of "\n"?
My comment rely on tests you have been made for example for `tprint("")` i think it should be more sufficient if we print nothing instead of blank line.

Reply:
`text2art` function return `\n` if the next word is available (`next_word == True`)